sql/core/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -285,7 +285,7 @@ object QueryTest extends Assertions {
       df: DataFrame,
       expectedAnswer: Seq[Row],
       checkToRDD: Boolean = true): Option[String] = {
-    val isSorted = df.logicalPlan.collect { case s: logical.Sort => s }.nonEmpty
+    val isSorted = df.logicalPlan.collectFirst { case s: logical.Sort => s }.nonEmpty
```

Comment:
how about `df.logicalPlan.exists(_.isInstanceOf[logical.Sort])`

Reply:
Thanks for offering a new idea, maybe it's better to stay the uniffle style in this pr, since the computational complexity is the same